esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -105,6 +105,28 @@ void CFileSprayEx::init(IPropertyTree *cfg, const char *process, const char *ser
     
     xpath.clear().appendf("Software/EspProcess[@name=\"%s\"]/EspService[@name=\"%s\"]/QueueLabel", process, service);
     cfg->getProp(xpath.str(), m_QueueLabel);
+    StringArray qlist;
+    getDFUServerQueueNames(qlist, nullptr);
+    if (qlist.ordinality())
+    {
+        if (!m_QueueLabel.length())
+            m_QueueLabel.append(qlist.item(0));
+        else
+        {
+            bool found = false;
```

Comment:
I agree probably not worth it for this PR, you could add a find option for case insensitivity to StringArray (there are similar already in the sort funcs.)